sampler_test.go

```diff
@@ -280,3 +288,55 @@ func TestGauge(t *testing.T) {
 		})
 	}
 }
+
+func TestTime(t *testing.T) {
+
```

Comment:
I would define the type inside here, since it is only used here. And would call testcase instead of metriccase...metriccase is a little confusing...at least for me.